workers/ohsome_quality_analyst/cli/cli.py

```diff
@@ -82,7 +82,13 @@ def list_fid_fields():
 def get_available_regions():
     """List available regions."""
     regions = asyncio.run(db_client.get_available_regions())
-    click.echo(regions)
+    output = []
+    header = "ID,Name,"
+    for feature in regions["features"]:
+        string = str(feature["id"]) + "," + feature["properties"]["name"] + ","
```

Comment:
I suggest using the build in library `csv` for this taks. See CSV File Reading and Writing docs: https://docs.python.org/3/library/csv.html